crates/cast/src/main.rs

```diff
@@ -321,6 +350,7 @@ async fn main() -> Result<()> {
             print_command_result("show-config", &mut result, value_format, cli.json)?;
             Ok(())
         }
+        Commands::Script(_) => Err(anyhow!("All cases should already be handled")),
```

Comment:
Why don't we call `script::run` here?

Reply:
Discussed here https://github.com/foundry-rs/starknet-foundry/pull/1067#discussion_r1392461269